docs/develop/run.md

```diff
@@ -300,7 +300,7 @@ curl -k -X POST https://localhost:5000/api/rdm-records/jnmmp-51n47/draft/actions
 And then search for it:
 
 ``` bash
-curl -k -XGET https://localhost:5000/api/records/?q=Gladiator | python3 -m json.tool
+curl -k -XGET https://localhost:5000/api/rdm-records?q=Gladiator | python3 -m json.tool
```

Comment:
Nice, I didn't manage to do another round of doc review, so that seems to cover all these tricky details.